pkg/apis/eksctl.io/v1alpha5/types.go

```diff
@@ -749,6 +751,8 @@ type Operator struct {
 	Namespace string `json:"namespace,omitempty"` // e.g. flux
 	// +optional
 	WithHelm *bool `json:"withHelm,omitempty"` // whether to install the Flux Helm Operator or not
+	// +optional
+	ReadOnly bool `json:"readOnly,omitempty"` // Instruct Flux to read-only mode and create the deploy key as read-only
```

Comment:
Let's move this into the `Operator` field

Reply:
I guess you've misread it, by any chance? It's already a field of the `Operator` struct, if that's what you mean. Please feel free to correct me if I'm missing something.

Reply:
you are right! I misread this, sorry about that